backend/test/support/fixtures/devices_fixtures.ex

```diff
@@ -117,18 +117,25 @@ defmodule Edgehog.DevicesFixtures do
   Generate a system_model.
   """
   def system_model_fixture(opts \\ []) do
```

Comment:
```suggestion
  def system_model_fixture(opts) do
```
We shouldn't define `[]` as the default value for `opts`, `opts` must be a non-empty keyword list with (at least) the `:tenant` key.

Reply:
The problem (and reason I'm providing `opts` with a default value) is that if you do that you get:
```
error: undefined function system_model_fixture/0 (expected Edgehog.BaseImagesTest to define such a function or for it to be imported, but none are available)
  test/edgehog/base_images_test.exs:35: Edgehog.BaseImagesTest.__ex_unit_setup_0_0/1
[...]
```
and other errors, because unfortunately even if tests are excluded with `--only`, they still all get compiled and an undefined function is a compilation error, and I don't want to fix all calling code in tests that are probably going to be removed anyway.

I plan on removing all default opts at the end of the porting.